plans/admin.py

```diff
@@ -151,8 +157,8 @@ class UserPlanAdmin(UserLinkMixin, admin.ModelAdmin):
     list_display_links = list_display
     list_select_related = True
     readonly_fields = ['user_link', ]
-    fields = ('user_link', 'plan', 'expire', 'active' )
```

Comment:
Any reason why they are being converted to lists ?

Reply:
Changed back in #80